docs/executor/index.rst

```diff
@@ -36,6 +36,9 @@ the example below.
     $ airflow config get-value core executor
     SequentialExecutor
 
+Supported backends
+^^^^^^^^^^^^^^^^^^
```

Comment:
Very minor comments:

Shouldn't this be a subsection (-) rather than subsubsection (^)?

And given it's a section name, let's have "Backends"?

Reply:
From Sphinx docs:

> Normally, there are no heading levels assigned to certain characters as the structure is determined from the succession of headings.

Unless we can force it with a pre-commit check and It doesn't affect the final result, I don't care what specific character. The user's perspective is important to me.  Our environment is too open for this rule to be enforced in any other way. There will still be changes to the documentation that do not follow the convention. 
Before structure: 
<img width="389" alt="Screenshot 2020-08-19 at 12 57 08" src="https://user-images.githubusercontent.com/12058428/90626673-9204bd80-e21b-11ea-938c-995d1efa2d33.png">

And when it comes to case, it is also not standardized in this project. Elsewhere in the documentation, we have a similar lowercase header.
<img width="524" alt="Screenshot 2020-08-19 at 12 57 23" src="https://user-images.githubusercontent.com/12058428/90626870-ba8cb780-e21b-11ea-81c2-da5abe0cae32.png">
I will gladly correct it here.